doc/manual/src/contributing/testing.md

```diff
@@ -0,0 +1,194 @@
+# Running tests
```

Comment:
I encountered the same thing just yesterday. I'd quickly merge moving that section out into its own page if you split out a PR for that.